app/views/divisions/history.html.haml

```diff
@@ -2,14 +2,16 @@
   = Diffy::CSS
 
 - content_for :title do
-  Division Description Changes —
-  = @division.original_name
-  —
-  = formatted_date(@division.date)
+  Division Description Changes
```

Comment:
I don't think this needs to be nested in a `do` block any more so you can inline it into the `content_for` method call.